BossMod/Autorotation/Standard/AkechiGNB.cs

```diff
@@ -12,7 +12,6 @@ public enum PotionStrategy { Manual, AlignWithRaidBuffs, Immediate } //Pots stra
     public enum LightningShotStrategy { OpenerRanged, Opener, Force, Ranged } //RangedUptime strategy
     public enum CartridgeStrategy { Automatic, Delay, ForceDD, ForceGF, ForceBS, ForceFC, ForceAll } //Cartridge usage strategy
     public enum OffensiveStrategy { Automatic, Force, Delay } //CDs strategies
-    public enum TrajectoryStrategy { Automatic, Forbid, Force, GapClose } //GapCloser strategy
```

Comment:
I assume GNB's gapcloser has no potency, so is not used as part of rotation? in such case, this is a correct change, yes.

Reply:
Correct.